Styling/README.md

```diff
@@ -598,6 +620,18 @@ Returns the arctangent value of the number (in radians).
 }
 ```
 
+#### atan2
+
+`atan2(y : Number, x : Number) : Number`
+
+Returns the arctangent of the quotient of `y` and `x` (in radians).
+
+```json
+{
+    "show" : "atan2(${Angle}, ${temperature}) > 0.0"
```

Comment:
I would use different variable names here. `GridY` and `GridX` might be good names.